suite-native/accounts/src/components/AccountListItem.tsx

```diff
@@ -49,9 +45,7 @@ export const valuesContainerStyle = prepareNativeStyle(utils => ({
 
 export const AccountListItem = ({ account, areTokensDisplayed = false }: AccountListItemProps) => {
     const { applyStyle } = useNativeStyles();
-    const accountLabel = useSelector((state: AccountsRootState) =>
-        selectAccountLabel(state, account.key),
-    );
+    const { accountLabel } = account.metadata;
```

Comment:
The selector is unnecessary in this file but is needed in another filed where the account object is not present. But you are right that the selector itself is unnecessarily complicated. I refactored the selector in https://github.com/trezor/trezor-suite/pull/8670/commits/3607920b6bad1b440dc450b4e1353d433122d8ba

Reply:
Yeah, I understand, but point of selectors is not only to have memoization etc, but also unify selecting state pieces. I would still like to use the selector here so we can only change selector without having to make changes to this file as well when data structure changes.

Reply:
I agree but make sense to me only if you modify the content of the selected attribute (e.g if we would making the `accountLabel` upperCase in this case).

We are accessing objects attributes received from selector all over the codebase. eg we are selecting an account and then directly  accessing `account.symbol` without use of a new selector for an accountSymbol:
```ts
const account = useSelector((state: AccountsRootState) =>
        selectAccountByKey(state, accountKey),
    );
    
   ....
  ... account.symbol ...
```

Why in this case it is ok, but in case of `accountLabel` it is not?